### PR TITLE
Cm 939

### PR DESCRIPTION
**Pull request by rodriguesk**

### CODE REVIEW

File: src/pages/userB/EditUserB/UserBEditNameForm.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```
Comment (NickCallaghan) [inline src/pages/userB/EditUserB/UserBEditNameForm.tsx:1]: This is a component not a page, can you move it under the componenets directory

File: src/api/putUserBNameConversation.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { climateApi } from "./apiHelper";
```
Comment (NickCallaghan) [inline src/api/putUserBNameConversation.ts:1]: we already have an api call in place for this, use `updateOneConversation`

File: src/api/postNewName.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { climateApi } from '../api/apiHelper';
```
Comment (NickCallaghan) [inline src/api/postNewName.ts:1]: we already have an api call in place for this, use updateOneConversation

File: src/hooks/useNewName.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { useMutation } from 'react-query';
```
Comment (NickCallaghan) [inline src/hooks/useNewName.tsx:1]: we already have a hook to do this, use `useUpdateConversation`

File: src/contexts/userBEdit.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React, { createContext, useState } from 'react';
```
Comment (NickCallaghan) [inline src/contexts/userBEdit.tsx:1]: I don't think we need context here are it's only the edit form component that need to know about this. Will be more simple to deal with in the component. `<UserBEditForm/>`

File: src/api/putUserBNameConversation.ts (hunk 2 above)
Comment (christianalafaa) [reply src/api/putUserBNameConversation.ts]: Alright, noted

File: src/api/postNewName.ts (hunk 3 above)
Comment (christianalafaa) [reply src/api/postNewName.ts]: Done

File: src/contexts/userBEdit.tsx (hunk 5 above)
Comment (christianalafaa) [reply src/contexts/userBEdit.tsx]: Alright, noted!

File: src/pages/userB/EditUserB/UserBEditNameForm.tsx (hunk 1 above)
Comment (christianalafaa) [reply src/pages/userB/EditUserB/UserBEditNameForm.tsx]: Done!

File: src/hooks/useNewName.tsx (hunk 4 above)
Comment (christianalafaa) [reply src/hooks/useNewName.tsx]: Alright, noted

**Comment by christianalafaa** [comment]
Done all the changes as requested.

**Review by NickCallaghan — DISMISSED** [review]

**Pull request merged**